### ISSUE TITLE
Add guided tutorial

### ISSUE THREAD

**Issue by eddycharly** [kyverno/playground#115]
It would be awesome to have a guided tutorial, perhaps in place of the "Examples" button (since this does the same thing as opens the "hamburger" menu on the left).

This guided tutorial or wizard would walk the user through the UI and explain what each section does. Even guides them on exploring their first sample, modifying a resource, modifying the policy, and testing. This would be awesome if done in a really nice way!

**Comment by fjogeleit** [comment]
@chipzoller text can be changed here: https://github.com/kyverno/playground/blob/main/frontend/src/composables/onboarding.ts

you can also provide the text and I update it - as you like

**Comment by eddycharly** [comment]
@fjogeleit shall we close this ?

**Comment by fjogeleit** [comment]
@chipzoller wanted to improve the text and I thought about adding the load and save button to the guide

**Comment by eddycharly** [comment]
Ok

**Comment by chipzoller** [comment]
Yes, I have a lot of work to do here but need to finish 1.10 release first :)
